Libplanet/Blockchain/Renderers/IRenderer.cs

```diff
@@ -0,0 +1,132 @@
+#nullable enable
+using System;
+using Libplanet.Action;
+using Libplanet.Blocks;
+
+namespace Libplanet.Blockchain.Renderers
+{
+    /// <summary>
+    /// Listens state changes on a <see cref="BlockChain{T}"/>.
+    /// <para>Usually, implementations of this interface purpose to update the in-memory game states
+    /// (if exist), or send a signal to the UI thread (usually the main thread) so that the graphics
+    /// on the display is redrawn.</para>
+    /// </summary>
+    /// <typeparam name="T">An <see cref="IAction"/> type.  It should match to
+    /// <see cref="BlockChain{T}"/>'s type parameter.</typeparam>
+    public interface IRenderer<T>
+        where T : IAction, new()
+    {
+        /// <summary>
+        /// Does things that should be done right after an <paramref name="action">action</paramref>
```

Comment:
I believe it can be presented like below:

```suggestion
        /// Does things that should be done right after an <paramref name="action"/>
```

Or is there another intention?

Reply:
Indeed no other intention, I am going fix it.

Reply:
@moreal Amended!